tests/cmdline/commands/test_config.py

```diff
@@ -7,143 +7,162 @@
 # For further information on the license, see the LICENSE.txt file        #
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
-# pylint: disable=no-self-use
-"""Tests for `verdi config`."""
-import pytest
-
+"""Tests for ``verdi config``."""
 from aiida import get_profile
 from aiida.cmdline.commands import cmd_verdi
-from aiida.manage.configuration import get_config
 
 
-class TestVerdiConfig:
-    """Tests for `verdi config`."""
+def test_config_set_option_no_profile(run_cli_command, empty_config):
+    """Test the `verdi config set` command when no ."""
+    config = empty_config
 
-    @pytest.fixture(autouse=True)
-    def setup_fixture(self, config_with_profile_factory):
-        config_with_profile_factory()
+    option_name = 'daemon.timeout'
+    option_value = str(10)
 
-    def test_config_set_option(self, run_cli_command):
-        """Test the `verdi config set` command when setting an option."""
-        config = get_config()
+    options = ['config', 'set', option_name, str(option_value)]
+    run_cli_command(cmd_verdi.verdi, options)
```

Comment:
If no profile is present, this defaults to setting on the `--global` level?
This makes sense to me as a policy, just checking.

Reply:
Indeed, if there are no profiles all option settings will always operate on global scope